app/src/main/res/values-az/strings.xml

```diff
@@ -31,7 +31,7 @@
     <string name="account_settings_subscribed">Abunə olundu</string>
     <string name="addBookmark">Əlfəcinlərə əlavə et</string>
     <string name="bottomBar_bookmarks">Əlfəcinlərimə keç</string>
-    <string name="app_bars_dot_spacer">·</string>
+    <string name="app_bars_dot_spacer" translatable="false">·</string>
```

Comment:
Almost there, now remove this line here and add it in values/strings.xml (replace the old line there)